cpp/include/cudf/io_types.h

```diff
@@ -64,9 +64,13 @@ typedef struct {
 
   bool			skip_blank_lines;			// whether or not to ignore blank lines
 
-  char			**true_values;				// array of char *	what should be considered as True - each char string contains {col ID, true value, ...} - this will allow multiple true values to be specified over multiple columns
-  char			**false_values;				// array of char *	what should be considered as True - each char string contains {col ID, false value, ...} - this will allow multiple false values to be specified over multiple columns
-  char			**na_values;				// array of char *	what should be considered as True - each char string contains {col ID, na value, ...} - this will allow multiple na values to be specified over multiple columns
+  const char	**true_values;				/**< list of values to recognize as boolean True */
```

Comment:
Please make indentation match. Tabs are terrible for this. See how the variable names and comments don't line up when you view it on github?

Reply:
Yeah, personally, I don't like using tabs because of this. I was trying to match the existing code which uses tabs of size 4. GitHub uses a different tab size so that's why it looks all unaligned. If I change it for one, it looks terrible for the other.